### DISCUSSION TITLE
Use NEUTRAL for console test results which have 0 points instead of FAILURE

### DISCUSSION THREAD

**Discussion by taylor** [lfn-cnti/testsuite#519] [Ideas]
**Is your feature request related to a problem? Please describe.**

Currently if a test is not applicable or if a test should not have negative results when it does not succeed it gets 0 points which is expected, as the test not passing does not indicate a negative issue.   What is unexpected for end users looking at the console output is seeing **FAILURE** for those tests.  A more accurate word should be used. 

An example would be checking for the use of K8s Secrets. If an application needs no confidential information stored for application configuration there is no need to use them and the test should be neutral in scoring. Giving this test 0 points does that. Showing FAIL on the console when this test runs communicates inaccurate information.

For SPF authentication in DNS **neutral** is used to indicate that the result is not negative. See 
https://dmarcly.com/blog/why-spf-authentication-fails-none-neutral-fail-hard-fail-soft-fail-temperror-and-permerror-explained

**Describe the solution you'd like**

Update console results output to show **NEUTRAL** instead of **FAILURE** when a test result receives 0 points.

**Describe alternatives you've considered**

Other words:
- NA, N/A NOT APPLICABLE
- INVALID
- SKIPPED
- IGNORED

**Additional context**

![image](https://user-images.githubusercontent.com/26697/101676047-3b1c3180-3a20-11eb-9eaf-8e105acfc3d2.png)

**Comment by agentpoyo** [comment]
:+1:  I like the idea of just NEUTRAL if the test has 0 points but fails. Displaying FAILURE indicates it's a requirement and you should lose points.

IGNORED would be my runner up choice if there's 0 points and the test is not found.

**Reply by wvwatson** [reply]
Seems like NEUTRAL would be a good addition to the grading

**Comment by williscool** [comment]
I like it only thing I could come up with was `UNSCORED`

but I think neutral is good.

I also think it should be gray in color like how PASS is green and so on
